getting-started/migration-guide.mdx

```diff
@@ -112,6 +112,39 @@ you could [enable caching on Edgee](/services/performance/caching).
   Note: Edgee will still track events (client-side) if it detects that HTML pages are being cached by your CDN.
 </Note>
 
+### Case 4: Edgee is an additional call within your workflow
+
+Edgee is placed alongside your CDN, your origin server still receives requests directly from the CDN.
+
+
+<Frame caption="Case 4: Edgee as an additional step in your workflow">
+  <img
+    src="/images/migration-guide-infra-case-4.jpg"
+    alt="Case 4: Edgee behind a CDN"
+  />
+</Frame>
+
+#### Why and when it makes sense
+
+This case makes sense when you want additional security control and already have a CDN that handles DNS resolution and DDoS protection.
+
+Edgee can sit alongside your CDN ensuring you maintain full control over DNS resolution and TLS termination, with minimal infrastructure changes and no CDN chaining.
+
+
+#### What needs to happen
```

Comment:
This section renders a bit weird 🤔 

<img width="768" alt="Screenshot 2025-07-08 at 23 19 46" src="https://github.com/user-attachments/assets/555efe30-ce1a-4b1f-900d-1efec39125c6" />

Not a big deal, but we could simplify a bit to improve its readability.